staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -256,27 +297,25 @@ func (cfgCtl *configController) processNextWorkItem() bool {
 	return true
 }
 
-// syncOne attempts to sync all the API Priority and Fairness config
-// objects.  It either succeeds and returns `true` or logs an error
-// and returns `false`.
-func (cfgCtl *configController) syncOne() bool {
+func (cfgCtl *configController) SyncOne() (bool, bool, bool, time.Duration) {
```

Comment:
godoc telling me what the return values are.  something, something, retry, waitTime?

Reply:
Added.